community/kernel/src/main/java/org/neo4j/kernel/GraphDatabaseQueryService.java

```diff
@@ -41,6 +41,10 @@
     Node createNode( Label... labels );
     Node getNodeById(long id);
     Relationship getRelationshipById(long id);
+
     InternalTransaction beginTransaction( KernelTransaction.Type type, AccessMode accessMode );
+
+    InternalTransaction beginTransaction( KernelTransaction.Type implicit, AccessMode accessMode, long timeout );
```

Comment:
With durations it's always nice to know which unit they're in. I just dislike scurrying around in implementation to figure that out. Put in javadocs?